packages/core/src/scenes/TimeEvents.ts

```diff
@@ -109,6 +111,16 @@ export class TimeEvents {
    * @internal
    */
   public register(name: string): number {
+    if (!this.collisionLookup.has(name)) {
```

Comment:
This might be personal preference, but how about inverting this condition and instead return early if the name has already been used? That would allow us to get rid of one level of indentation since we can get rid of the `else` branch.

```suggestion
    if (this.collisionLookup.has(name)) {
      this.scene.project.logger.error({
        message: `name "${name}" has already been used for another event name.`,
        stack: new Error().stack,
      });
      return 0;
    }
 
    this.collisionLookup.add(name);
 
 // rest...
```

Reply:
Good suggestion, I would agree to change it to use one less indentation.

Reply:
I agree